examples/vg-specs/bar_grouped.vg.json

```diff
@@ -264,20 +264,11 @@
                     },
                     "encode": {
                         "update": {
-                            "width": {
-                                "signal": "child_width"
-                            },
-                            "height": {
-                                "signal": "child_height"
-                            },
                             "stroke": {
                                 "value": "#ccc"
                             },
                             "strokeWidth": {
                                 "value": 0
-                            },
-                            "fill": {
```

Comment:
Any idea why this is happening? I don't see where I made a change that could cause this.